src/sentry/identity/oauth2.py

```diff
@@ -64,6 +65,9 @@ def _get_oauth_parameter(self, parameter_name):
     def get_oauth_access_token_url(self):
         return self._get_oauth_parameter('access_token_url')
 
+    def get_oauth_refresh_token_url(self):
```

Comment:
Had anticipated moving things in here, but couldn't figure out a clean way of doing so. Open to suggestions, otherwise I'll just remove this code.